projects/stone_cx/robotVicon.cc

```diff
@@ -43,21 +42,7 @@ int main(int argc, char *argv[])
     // Initialise GeNN
     allocateMem();
     initialize();
-
-    //---------------------------------------------------------------------------
-    // Initialize neuron parameters
-    //---------------------------------------------------------------------------
-    // TL
-    for(unsigned int i = 0; i < 8; i++) {
-        preferredAngleTL[i] = preferredAngleTL[8 + i] = (Parameters::pi / 4.0) * (double)i;
-    }
-
-    //---------------------------------------------------------------------------
-    // Build connectivity
-    //---------------------------------------------------------------------------
-    buildConnectivity();
-    
-    initstone_cx();
+    initializeSparse();
```

Comment:
Alright, this is *really* nit-picky, but we are using British spelling elsewhere so maybe it should be ``initialiseSparse`` for consistency

Reply:
Sadly GeNN has something of a mix but, in GeNN, initialisation occurs in two parts and the first part is triggered with an ``initialize`` function so it would seem like a bad idea to use the British spelling for the 2nd one...

Reply:
Ahh no worries then. Let's be consistent with GeNN.

Reply:
To be clear, that **is** a call into GeNN-generated code

Reply:
Ohh I see.